hera_cal/tests/test_utils.py

```diff
@@ -611,6 +611,9 @@ def test_red_average():
     d = np.asarray([hd.get_data(bl + ('ee',)) for bl in reds[0]])
     w = np.asarray([(~hd.get_flags(bl + ('ee',))).astype(float) for bl in reds[0]])
     davg = np.sum(d * w, axis=0) / np.sum(w, axis=0).clip(1e-10, np.inf)
+    # set flagged data to 1.0+0.0j
```

Comment:
the consequence of changing that line in `utils.red_average` is a need to update this unit test, which seems fairly isolated to this specific functionality and wouldn't affect other parts of hera_cal.